external/boost/CMakeLists.txt.boost.in

```diff
@@ -20,12 +20,69 @@ cmake_minimum_required(VERSION 3.9)
 project(boost-download NONE)
 
 include(ExternalProject)
-ExternalProject_Add(Boost
-  URL               @OLP_SDK_CPP_BOOST_URL@
-  DOWNLOAD_DIR      "${CMAKE_CURRENT_BINARY_DIR}/download"
-  SOURCE_DIR        "${CMAKE_CURRENT_BINARY_DIR}/external_boost"
-  CONFIGURE_COMMAND ""
-  BUILD_COMMAND     ""
-  INSTALL_COMMAND   ""
-  TEST_COMMAND      ""
+
+if (WIN32)
+    set(BOOTSTRAP_CMD "bootstrap.bat")
+    set(B2_CMD "b2.exe")
+else()
+    if (CMAKE_CXX_COMPILER_ID STREQUAL "AppleClang")
+        set(BOOTSTRAP_CMD "./bootstrap.sh --with-toolset=clang-darwin")
+    else()
+        set(BOOTSTRAP_CMD "./bootstrap.sh")
+    endif()
+    set(B2_CMD "./b2")
+endif()
+
+ExternalProject_Add(boost-download
+    GIT_REPOSITORY      @OLP_SDK_CPP_BOOST_URL@
+    GIT_TAG             boost-1.72.0 
+    GIT_SUBMODULES      libs/any
```

Comment:
Do we need all?

Reply:
Yes, remove one, and something will not be found